### ISSUE TITLE
xslTNG fails because of oxy-markup after column deletion in file

### ISSUE THREAD

**Issue by frank-steimke** [docbook/xslTNG#312]
Hi,

we are testing xslTNG 2.1.2 in Oxygen (25.1 on windows 10). Results are promising. We use the Stylesheets for a  large book (approx. 400 pages).

Today i ran a Document comparison of two versions of this Book with _deltaXML DocBook Compare_. The result is a large Document with lots of oxy-markup (attached). 
[changes.zip](https://github.com/docbook/xslTNG/files/11334667/changes.zip)

xslTNG fails when i try to produce an HTML Version from that Document. I am pretty shure that the reason is the deletion of a column in an informaltable, which is of course detected by deltaXML and leads to a difficult structure of change-tracking.

I admit that this a really difficult document, but on the other hand, it is real life.

Thanks in advance,
Frank Steimke

**Comment by ndw** [comment]
Fails how? I'll give it a try when I have a chance.

**Comment by frank-steimke** [comment]
Sorry. Terminates with error message _Overlapping cells in table at row 10 for column 3_.

**Comment by ndw** [comment]
Yes, the code that works out how to transform the table doesn't know anything about change markup. If the markup makes a table with three actual columns that claims to be two columns wide, that's going to cause the stylesheets to fail.

I'm not sure it's going to be practical to try to integrate the change markup into table processing, but I can add an option to make the overlapping cells message not stop the processing. You may get gibbersh table transformations, though.

**Comment by ndw** [comment]
I looked into the possibility of allowing the CALS table markup to create a "broken" table in this case, but it's impractical. CALS table processing is complicated and the code relies on the structural correctness of the table.

Rather than simply discarding the delted table cells, I've added a note to the output where the deleted cells would have been. If, for example, the cell in column 2 would have been followed by another "deleted" cell, the cell in column 2 includes a note to the effect that a delted following cell is not shown.

Hopefully that's sufficient.
